src/notification.rs

```diff
@@ -59,6 +59,10 @@ macro_rules! lsp_notification {
     ("workspace/didChangeWorkspaceFolders") => {
         $crate::notification::DidChangeWorkspaceFolders
     };
+    // Proposed
+    // ("window/progress") => {
```

Comment:
Initially I wanted to warn the user if they're trying to use a feature that's proposed but since the macro has to return specifically a type, I wasn't sure what to return here, so I just commented it out

Reply:
Should be fine to just return the type as normal, if `proposed` is not enabled then it will just be a compile error. (Add a comment on it so people can at least read to source to see why it does not work).